html/template.shtml

```diff
@@ -88,10 +88,12 @@
         <!--#include file="partials/$partial.shtml" -->
         <a href="javascript: window.history.back()" title="Back" class="button">Back</a>
         <a href="/" title="Start again" class="button">Start again</a>
-        <p style="color: #6F777B;">
-          Generated at <!--#config timefmt="%H:%M:%S %Z" --><!--#echo var="date_local" -->
-          on <!--#config timefmt="%A, %d %B %Y" --><!--#echo var="date_local" -->.
-        </p>
+        <!--#if expr="$verbose_error_pages = TRUE" -->
+          <p style="color: #6F777B;">
+            Generated at <!--#config timefmt="%H:%M:%S %Z" --><!--#echo var="date_local" -->
+            on <!--#config timefmt="%A, %d %B %Y" --><!--#echo var="date_local" -->.
+          </p>
+        <!--#endif -->
       </div>
```

Comment:
Sure you want to remove this?

Reply:
if non-debug yes, my user/content researcher/designer wants it gone